### PR TITLE
Makefile: add test target for applying changes

### PR DESCRIPTION
**Pull request by aalemayhu**
retyping all of these commands is very tedious.

Signed-off-by: Alexander Alemayhu <alexander@alemayhu.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -92,5 +92,14 @@ generate-api:
 vps:
 	VBoxManage list runningvms
 
+tt:
+	cd daemon && make go-bindata
+	make
+	sudo systemctl stop cilium
```
Comment (aanm) [inline Makefile]: You need to stop and start `cilium-docker` as well.

**Review by aanm — CHANGES_REQUESTED** [review]

File: Makefile [hunk 2]
```diff
@@ -92,5 +92,14 @@ generate-api:
 vps:
 	VBoxManage list runningvms
 
+tt:
```
Comment (tgraf) [inline Makefile]: I'm fine with this but let's give this a better name. `make tt` is not very descriptive.

Comment (aalemayhu) [reply Makefile]: Was hoping to keep it short, how about one of these?

`make reload`
`make change`
`make apply`

Comment (tgraf) [reply Makefile]: I'm happy with `make reload`. You can always have an alias for you locally if you like `make tt`

**Review by tgraf — APPROVED** [review]

**Review by aanm — APPROVED** [review]

**Pull request merged**